requirements.txt

```diff
@@ -20,4 +20,4 @@ soundfile
 # Miscellaneous scripts
 paramiko >= 2.1
 scipy
-matplotlib
+matplotlib
```

Comment:
there's no change here

Reply:
This is removing the newline at the end of the file. Please add it back.